app/move/index.js

```diff
@@ -39,13 +54,28 @@ router.use(
   wizard(createSteps, createFields, createConfig)
 )
 router.get('/:moveId', protectRoute('move:view'), view)
+
 router.get('/:moveId/confirmation', protectRoute('move:create'), confirmation)
 router.use(
   '/:moveId/cancel',
   protectRoute('move:cancel'),
   wizard(cancelSteps, cancelFields, cancelConfig)
 )
 
+updateSteps.forEach(updateStep => {
+  const key = Object.keys(updateStep)[0]
+  const updateStepConfig = {
+    ...updateConfig,
+    name: 'update-a-move' + key,
+    journeyName: 'update-a-move' + key,
+  }
+  router.use(
+    '/:moveId',
```

Comment:
Wonder if it should include a clear url segment  as well to show a step is being edited: `/:moveId/edit`?

Reply:
I suppose that we might want to show only parts of a move, so your suggestion is probably a good and sensible one.